FerramAerospaceResearch/FARGUI/FAREditorGUI/Simulation/InstantConditionSim.cs

```diff
@@ -158,7 +158,7 @@ public void GetClCdCmSteady(InstantConditionSimInput input, out InstantCondition
                 if (!(w && w.part))
                     continue;
 
-                w.ComputeForceEditor(velocity.normalized, input.machNumber, 2);
+                w.ComputeForceEditor(velocity, input.machNumber, 2);
```

Comment:
`velocity` is no longer normalized here, this will affect force computations

Reply:
This question is not meant in a harsh way; I am just a bit clumsy writing it succinct.
Can you produce an example where the velocity.normalized is (strictly) closer to normalized than the current ('not normalized') velocity?
If so that may also be a good reason to use Vector3.cross for sideways, that you mentioned in an earlier comment.

Reply:
You're right, sorry. Had a mistake in my calculations. Still, the floating point errors can accumulate so I would normalize the vector afterwards anyway.

Reply:
Sure. We can do that to make it more easy to read the intention of the formulas.
(for what it is worth I suspect that the direct formulas uses double precision floating point math and the Normalize have to deal with single precision and won't improve anything).